app/controllers/DigitalPack.scala

```diff
@@ -14,23 +15,23 @@ object DigitalEdition {
 
 object DigitalPack extends Controller {
 
-  def uk = Action {
+  def uk = CachedAction {
     Ok(views.html.digitalpack.info(DigitalEdition.UK))
   }
 
-  def us = Action {
+  def us = CachedAction {
     Ok(views.html.digitalpack.info(DigitalEdition.US))
   }
 
-  def au = Action {
+  def au = CachedAction {
     Ok(views.html.digitalpack.info(DigitalEdition.AU))
   }
 
-  def selectCountry = Action {
+  def selectCountry = CachedAction {
     Ok(views.html.digitalpack.country())
   }
 
-  def handleCountry = Action(parse.tolerantFormUrlEncoded) { request =>
+  def handleCountry = CachedAction(parse.tolerantFormUrlEncoded) { request =>
```

Comment:
I don't think we should be caching the response to this POST request! Unless you're sure that Fastly is varying the response based on request body.